tests/ci_build/test_python.sh

```diff
@@ -26,34 +34,40 @@ function install_xgboost {
   fi
 }
 
+function uninstall_xgboost {
+  pip uninstall -y xgboost
+}
+
 # Run specified test suite
 case "$suite" in
   gpu)
     source activate gpu_test
     install_xgboost
-    pytest -v -s -rxXs --fulltrace -m "not mgpu" tests/python-gpu
+    pytest -v -s -rxXs --fulltrace -m "not mgpu" ${args} tests/python-gpu
+    uninstall_xgboost
```

Comment:
Why do you have to install xgboost?

Reply:
This is because XGBoost is built in the previous step of the Jenkins pipeline. So here we only get access to a Python wheel.